packages/pluggableWidgets/time-series-chart-web/src/TimeSeries.xml

```diff
@@ -107,6 +107,8 @@
                                     <enumerationValue key="last">Last</enumerationValue>
                                 </enumerationValues>
                             </property>
```

Comment:
You should also move `staticTooltipHoverText and `dynamicTooltipHoverText to `General`, after `Aggregation function`: https://github.com/mendix/widgets-resources/pull/1352/files#diff-86639a8728fc318227f72935ee08ffc1b3185a149f79c783b437987715cb308fR118